src/app/public/styles/ag-grid-styles.scss

```diff
@@ -221,15 +236,26 @@ ag-grid-angular.sky-ag-grid-editable {
 
   /* fine tune focus cell borders */
   .ag-row {
-    .ag-cell-focus {
-      line-height: ag-param(row-height) - 4;
+    &.ag-row-focus {
+      z-index: 1;
     }
 
-    .ag-cell-focus.ag-cell-inline-editing {
-      height: ag-param(row-height) - 1;
-      padding-left: 0;
-      padding-right: 0;
-      @include sky-field-status(active)
+    .ag-cell.ag-cell-focus {
+      line-height: ag-param(row-height) - 4;
+      z-index: 1;
+
+      &.ag-cell-inline-editing,
+      &.ag-cell-popup-editing {
+        height: ag-param(row-height) - 1;
+        padding-left: 0;
+        padding-right: 0;
+        border: 2px solid $sky-theme-modern-color-blue-74;
```

Comment:
can use `$sky-theme-modern-border-color-primary-dark` instead of the direct color token (i see some other cases of this in the css that can also be replaced)